Makefile

```diff
@@ -70,6 +81,20 @@ ifneq ($(USE_GOCO),)
 	GOFMT_CMD:=$(CONTAINER_CMD) run $(CONTAINER_OPTS) --rm $(GOCACHE_VOLUME) -v $(CURDIR):/go/src/github.com/ceph/go-ceph$(VOLUME_FLAGS) --entrypoint $(GOFMT_CMD) $(CI_IMAGE_TAG)
 endif
 
+# Assemble the various build args that will be passed container build command(s)
+CONTAINER_BUILD_ARGS:=$(DEFAULT_BUILD_ARGS)
```

Comment:
I wanted to avoid having a variable that got mutated. IOW, `DEFAULT_BUILD_ARGS` is always what you the user supplied. Then `CONTAINER_BUILD_ARGS` can then get mutated later on.  It's largely born from some frustration I had getting the correct combintation of vars to work when trying to test out minor tweaks to the command line that calls the container build. Too many times I'd try to "add" an argument to mix only to end up overriding *everything* in the makefile that updated that var.  This way, if you simply use `make DEFAULT_BUILD_ARGS="-my-extra-stuff"` it will tend to "just work"... without me having to think hard about it. :-)